packages/databricks-vscode/resources/python/00-databricks-init.py

```diff
@@ -357,6 +359,99 @@ def df_html(df):
     html_formatter.for_type(SparkConnectDataframe, df_html)
     html_formatter.for_type(DataFrame, df_html)
 
+@logErrorAndContinue
```

Comment:
This will show a message saying notebook init failed. We can skip showing that message by not logging error here.